sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableReadIT.java

```diff
@@ -41,8 +42,10 @@ public void testE2EBigtableRead() throws Exception {
     BigtableTestOptions options = TestPipeline.testingPipelineOptions()
         .as(BigtableTestOptions.class);
 
+    String project = TestPipeline.testingPipelineOptions().as(GcpOptions.class).getProject();
```

Comment:
Just for clarity, this will re-initialize the options, so just `options.as(GcpOptions.class).getProject()` will do here.